index.html

```diff
@@ -15,11 +15,25 @@
             width: 100%;
             height: 100%;
         }
+        .controls {
+            position: absolute;
+            top: 40px;
+            left: 5px;
+            background-color: rgba(255, 255, 255, 0.5); 
+        }
```

Comment:
I'm not a web designer, so I just threw the buttons below the navigation controls and added a translucent background. The UI can be spiffed up later.